### PR TITLE
Major update

### PR DESCRIPTION
**Pull request by Miercko**

### CODE REVIEW

File: src/assets/scss/theme/_scaffolding.scss [hunk 1]
```diff
@@ -35,8 +35,8 @@ html {
 
 body {
   overflow-y: scroll;
-  padding-top: 70px;
-  padding-bottom: 30px;
+  //padding-top: 70px;
+  //padding-bottom: 30px;
```
Comment (mischah) [inline src/assets/scss/theme/_scaffolding.scss]: Bitte rausschmeißen anstatt auszukommentieren.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+          <p>
+             The aim of this project is to help you to build your things.
+             <br>
+             From Bootstrap themes over static websites to single page applications.
+          </p>
+          <div><a class="button" href="#anchor-install" >Install Now</a></div>
+    </div>
+ </div>
+ <div class="row our-patners gray">
+       <div class="patner bmbf col-lg-2 offset-lg-2 col-md-2 offset-md-1 col-ms-12 col-xs-12">
```
Comment (mischah) [inline src/index.html]: Die Klasse `patner` umbennen in `partner`.

Comment (mischah) [reply src/index.html]: Taucht an mehreren Stellen auf.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. Lorem ipsum dolor sit amet,
```
Comment (mischah) [inline src/index.html]: Blindtext ersetzen.

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+       <div class="patner offset-lg-1 offset-md-1 col-lg-2 col-md-2 col-ms-12 col-xs-12">
+          <img src="assets/img/DLR.png" alt="DLR">
+       </div>
+ </div>
+ <div class="row">
+    <div id="anchor-about" class="about white col-lg-12 col-md-12 col-ms-12 col-xs-12">
+       <h2>What advantages does Baumeister offer?</h2>
+       <p>
+          The administration of dependencies can be completely solved via the `package.json` of the project. No additional file needs to be modified, e.g. if new libraries or files like fonts need to be added to the JavaScript and CSS bundles. The package.json section `baumeister' contains the properties `bundleCSS`, `bundleExternalJS` and `includeStaticFiles` for this purpose.
+          <br><br>
```
Comment (mischah) [inline src/index.html]:
```
<p>
  <br><br>
</p>
```

ersetzen durch:

```
<p></p>
<p></p>
```

Das ist das semantischere Markup.

File: src/index.html [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+                </div>
+             </div>
+          </div>
+       </div>
+    </div>
+ </div>
+ <div class="row">
+    <div id="anchor-install" class="install gray col-lg-12 col-md-12 col-ms-12 col-xs-12">
+       <h2>Quick install guide</h2>
+       <h6>For those already using Node, Gulp and stuff.</h6>
```
Comment (mischah) [inline src/index.html]: Baumeister verwendet kein Gulp mehr. Von daher besser:

```suggestion
       <h6>For those already using Node and npm.</h6>
```

File: src/index.html [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+          <div class='wrp'>
+             <a target="_blank" rel="noopener noreferrer" href="https://twitter.com/baumeister_io" class='icon-twitter icon'>
+                <i class='fab fa-twitter'></i>
+                <p>Twitter</p>
+             </a>
+             <a target="_blank" rel="noopener noreferrer" href="https://github.com/micromata/baumeister" class='icon icon-github'>
+                <i class='fab fa-github'></i>
+                <p>Github</p>
+             </a>
+             <a target="_blank" rel="noopener noreferrer" href="https://labs.micromata.de/projects/baumeister.html" class='icon icon-labs'>
```
Comment (mischah) [inline src/index.html]: Url hat sich nach Labs Relaunch geändert.
```suggestion
             <a target="_blank" rel="noopener noreferrer" href="https://labs.micromata.de/baumeister/" class='icon icon-labs'>
```

File: src/index.html [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+$ npm run tasks
+       </code></pre>
+          <p>See https://github.com/<wbr>micromata/baumeister for details.</p>
+       </div>
+    </div>
+ </div>
+ <div class="row">
+    <div id="anchor-socials" class="keepintouch white col-lg-12 col-md-12 col-ms-12 col-xs-12">
+       <h2>Our Socials</h2>
+       <h6>Baumeister on...</h6>
```
Comment (mischah) [inline src/index.html]: Kleine typografische Verbesserung:

```suggestion
       <h6>Baumeister on …</h6>
```

File: src/index.html [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+          <p>See https://github.com/<wbr>micromata/generator-baumeister for details.</p>
+       </div>
+       <div class="viaGit">
+          <p>via <strong>Git</strong></p>
+          <pre class="language-bash"><code class="language-bash">$ <span class="token function">git</span> clone https://github.com/micromata/baumeister.git
+$ <span class="token function">cd</span> baumeister
+$ npm <span class="token function">install</span>
+$ npm run tasks
+       </code></pre>
+          <p>See https://github.com/<wbr>micromata/baumeister for details.</p>
```
Comment (mischah) [inline src/index.html]: Bitte verlinken.

File: src/index.html [hunk 9]
```diff
@@ -0,0 +1,240 @@
...
+       <h2>Quick install guide</h2>
+       <h6>For those already using Node, Gulp and stuff.</h6>
+       <div class="viaYeoman">
+          <p>via <strong>Yeoman</strong></p>
+          <pre class="language-bash"><code class="language-bash">$ npm <span class="token function">install</span> -g yo
+$ npm <span class="token function">install</span> -g generator-baumeister
+$ yo baumeister
+$ npm run tasks
+       </code></pre>
+          <p>See https://github.com/<wbr>micromata/generator-baumeister for details.</p>
```
Comment (mischah) [inline src/index.html]: Bitte verlinken.

File: src/index.html [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+          <img src="assets/img/DLR.png" alt="DLR">
+       </div>
+ </div>
+ <div class="row">
+    <div id="anchor-about" class="about white col-lg-12 col-md-12 col-ms-12 col-xs-12">
+       <h2>What advantages does Baumeister offer?</h2>
+       <p>
+          The administration of dependencies can be completely solved via the `package.json` of the project. No additional file needs to be modified, e.g. if new libraries or files like fonts need to be added to the JavaScript and CSS bundles. The package.json section `baumeister' contains the properties `bundleCSS`, `bundleExternalJS` and `includeStaticFiles` for this purpose.
+          <br><br>
+          Another advantage of Baumeister over other solutions is the simple adaptation of automation tasks through the maintainable structure. Each task is a separate ES6 module and in the Gulpfile these tasks are only plugged together. If a task is not needed during the entire automation process, it can therefore be localized very quickly and easily deactivated - even if you are not familiar with Baumeister's overall structure.
```
Comment (mischah) [inline src/index.html]: > Another advantage of Baumeister over other solutions is the simple adaptation of automation tasks through the maintainable structure. Each task is a separate ES6 module and in the Gulpfile these tasks are only plugged together. If a task is not needed during the entire automation process, it can therefore be localized very quickly and easily deactivated - even if you are not familiar with Baumeister's overall structure.


ersetzen durch:

> Another advantage of Baumeister over other solutions is the simple adaptation of automation tasks through the maintainable structure. Each task is a separate npm run script and in the package.json these tasks are only plugged together. If a task is not needed during the entire automation process, it can therefore be localized very quickly and easily deactivated - even if you are not familiar with Baumeister's overall structure.

File: src/assets/scss/_theme.scss [hunk 11]
```diff
@@ -0,0 +1,678 @@
...
+// --------------------------------------------------
+// @import "theme/testResponsiveHelpers"; // debug
+@import 'theme/footer';
+
+// Important note //
+// You could also use this file to insert theme related style definitions
+// directly within this file. But we recommend keeping your Sass code in
+// separate files like the examples above when you exceed a few hundred lines
+// of code. Otherwise it will definitely have a negative impact on
+// maintainability.
```
Comment (mischah) [inline src/assets/scss/_theme.scss:27]: Ähm, du hast dich irgendwie nicht an das gehalten, was da als Kommentar steht und wir besprochen hatten. Musst du jetzt aber auch nicht mehr unbedingt auseinander ziehen. Lass uns da kurz drüber sprechen.

**Review by mischah — CHANGES_REQUESTED** [review]
Du hast da auch noch Stylelint Fehler drin über die wir reden müssen 😗 

Ich wäre dann erstmal durch mit dem Review.

File: src/assets/scss/theme/_scaffolding.scss (hunk 1 above)
Comment (Miercko) [reply src/assets/scss/theme/_scaffolding.scss]: Done

File: src/index.html (hunk 2 above)
Comment (Miercko) [reply src/index.html]: Done

File: src/index.html (hunk 4 above)
Comment (Miercko) [reply src/index.html]: Done

File: src/index.html (hunk 5 above)
Comment (Miercko) [reply src/index.html]: Done

File: src/index.html (hunk 7 above)
Comment (Miercko) [reply src/index.html]: Done

File: src/index.html (hunk 8 above)
Comment (Miercko) [reply src/index.html]: Done

File: src/index.html (hunk 9 above)
Comment (Miercko) [reply src/index.html]: Done

File: src/index.html (hunk 10 above)
Comment (Miercko) [reply src/index.html]: Done

**Comment by mischah** [comment]
Wenn du Sachen auf »Done« hast, kannst du sie gerne direkt pushen.

File: src/index.html [hunk 12]
```diff
@@ -0,0 +1,241 @@
...
+
+	<!--
+	Replace "no-js" class with "js" in case JavaScript is available.
+	You can delete this when you feel the need to make use of modernizr.
+	-->
+	<script>(function(h){h.className = h.className.replace('no-js', 'js')})(document.documentElement)</script>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"></script>
+  <link href="https://use.fontawesome.com/releases/v5.0.4/css/all.css" rel="stylesheet">
```
Comment (mischah) [inline src/index.html:24]: Bitte nichts von einem CDN zu holen. 
Diese `link` und `script` Elemente können weg.

Bootstrap und jQuery liefert Baumeister von Hause aus.

fontawesome musst du via npm als dependency hinzufügen und dass CSS in unser Bundle aufnehmen
Siehe:
- https://github.com/micromata/baumeister#using-external-libraries
- https://github.com/micromata/baumeister#bundling-css-from-dependencies

File: src/index.html [hunk 13]
```diff
@@ -0,0 +1,241 @@
...
+       </p>
+    </div>
+ </div>
+ <div class="row">
+    <div id="anchor-iforp" class="iforp gray col-lg-12 col-md-12 col-ms-12 col-xs-12">
+       <h2>What offers iFORP</h2>
+       <p>
+        The collaborative, cross-disciplinary development of ideas is a very powerful, yet challenging approach to innovation. iForp supports this process with the rapid creation of WebApp prototypes. The prototype can be based on HTML/CSS/JSS as well as on images such as sketches, wireframes and screen designs. Individual views can be linked to logical clickflows. In addition, the tool supports the documentation of user feedback and design decisions. The tool is published under the MIT license (OSS) and runs as a webapp and can be installed and operated in your own infrastructure.
+       </p>
+       <div><a class="button" href="https://www.google.de" >Get iForp Now!</a></div>
```
Comment (mischah) [inline src/index.html]: Google Link ersetzen durch Link auf die Labs Seite. Siehe Profi Chat.

**Review by mischah — CHANGES_REQUESTED** [review]
Ich habe da noch was entdeckt wo du noch mal ran musst.

**Review by mischah — APPROVED** [review]
